### PR TITLE
ios 12 widget-related crash fix

### PR DESCRIPTION
**Pull request by benisgold**
Fixes RNBW-3185

## What changed (plus any additional context for devs)
- specify weak link to SwiftUI which is not compatible with iOS < 13
- can test on testflight build 1.6.12 (3)

## Final checklist

- [x] Assigned individual reviewers?
- [x] Added labels?
- [ ] Added e2e tests? if not please specify why
- [ ] If you added new files, did you update the CODEOWNERS file?

### CODE REVIEW

File: src/components/settings-menu/CurrencySection.js [hunk 1]
```diff
@@ -39,7 +40,10 @@ const CurrencySection = () => {
   const onSelectCurrency = useCallback(
     currency => {
       settingsChangeNativeCurrency(currency);
-      reloadTimelines('PriceWidget');
+      // reload widget timelines only if on ios version 14 or above
+      if (ios && parseInt(Platform.Version, 10) >= 14) {
```
Comment (osdnk) [inline src/components/settings-menu/CurrencySection.js:44]:
```suggestion
      if (ios && parseInt(Platform.Version) >= 14) {
```

this is not needed here AFAIK

**Review by osdnk — APPROVED** [review]
I cannot really check this as well!

**Comment by osdnk** [comment]
I think that realistically we don't need to wait to QA this. This is damn simple and there's no way this donna hurt.

**Comment by benisgold** [comment]
@osdnk i dowloaded the ios 12.4 emulators but i can't repro the issue

**Comment by ibrahimtaveras00** [comment]
Crash still seems to be happening on this testflight version 1.6.10 (10); seen here = http://recordit.co/f4ZmhzkLrc

**Comment by terrysahaidak** [comment]
Are we sure it's not native-related crash? Because it's crashing kinda immediately on the iphone 6. Too fast to be able to execute any of the JS code.

**Comment by benisgold** [comment]
i tested this build on testflight 1.6.12 (3) on ios 15 and ios 12

**Review by brunobar79 — APPROVED** [review]
👍 👍 👍

**Review by ibrahimtaveras00 — APPROVED** [review]
checked testflight 1.6.12 (3), all looks good; QA Passed 👍🏽

**Review by jkadamczyk — APPROVED** [review]

**Pull request merged**